atc/db/job.go

```diff
@@ -848,7 +847,7 @@ func scanJob(j *job, row scannable) error {
 	}
 
 	var config atc.JobConfig
-	err = yaml.Unmarshal(decryptedConfig, &config)
+	err = json.Unmarshal(decryptedConfig, &config)
```

Comment:
So was this the performance culprit?

Reply:
I think so... the pprof shows exactly that difference 
scanJob-> json.Unmarshal vs scanJob -> yaml.unmarshal

Reply:
oh ya, neat!

Reply:
Yep the fix itself is just this line. :+1: